modules/govuk/templates/asset_manager_extra_nginx_config.conf.erb

```diff
@@ -42,6 +43,10 @@ location ~ /cloud-storage-proxy/(.*) {
   # default, so we do not have to do that explicitly here.
   add_header ETag $etag_from_rails;
   add_header Last-Modified $last_modified_from_rails;
+  # Respect the optional Link header set by the Rails application,
+  # this optional header refers to the asset's owning document.
+  add_header Link $link_from_rails;
```

Comment:
What happens with this if the header *isn't* set by Rails?

Reply:
It's ignored safely by the looks of it...

```
irb(main):002:0> a.parent_document_url = nil
=> nil
irb(main):003:0> a.save
=> true
irb(main):004:0> exit
[vagrant@development asset-manager (master)]$ curl -I http://assets-origin.dev.gov.uk/media/5b45cd0a759b744a2df5cea0/tmp.txt
HTTP/1.1 200 OK 
Server: nginx
Date: Wed, 11 Jul 2018 10:00:14 GMT
Content-Type: text/plain
Content-Length: 29
Connection: close
Vary: Accept-Encoding
Vary: Accept-Encoding
Content-Disposition: inline; filename="tmp.txt"
Cache-Control: max-age=0, private, must-revalidate
Vary: Accept-Encoding
X-Request-Id: 8c282998-c5b7-4f60-aad7-51984c6c3feb
X-Runtime: 0.014815
ETag: "5b45cd0a-1d"
Last-Modified: Wed, 11 Jul 2018 09:25:30 GMT
X-Frame-Options: DENY
Strict-Transport-Security: max-age=31536000
Access-Control-Allow-Origin: *
Access-Control-Allow-Methods: GET, OPTIONS
Access-Control-Allow-Headers: origin, authorization
```

Reply:
I guess we want to really really make sure that Vary is set to Accept-Encoding.